### PR TITLE
#55 - seeders

### PR DESCRIPTION
**Pull request by MrHDOLEK**
- create seeders
- fix problem with startup
- fix problem for frontend (file extend)

Should close https://github.com/blumilksoftware/meetup/issues/55

### CODE REVIEW

File: database/seeders/FrontendSeeder.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            ->create();
+        $speakers = Speaker::all();
+        $organization = Organization::all();
+        foreach ($speakers as $speaker) {
+            Meetup::factory(10)
+                ->create([
+                    "user_id" => $user,
+                    "speaker_id" => $speakers->random(),
+                    "organization_id" => $organization->random(),
+                ]);
```
Comment (krzysztofrewak) [inline database/seeders/FrontendSeeder.php:38]:
```suggestion
        Organization::factory(10)->create();
        Speaker::factory(10)->create();
        $speakers = Speaker::all();
        $organization = Organization::all();
        foreach ($speakers as $speaker) {
            Meetup::factory(10)->create([
                    "user_id" => $user,
                    "speaker_id" => $speakers->random(),
                    "organization_id" => $organization->random(),
                ]);
```

File: database/seeders/FrontendSeeder.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            ->create();
+        Speaker::factory(10)
+            ->create();
+        $speakers = Speaker::all();
+        $organization = Organization::all();
+        foreach ($speakers as $speaker) {
+            Meetup::factory(10)
+                ->create([
+                    "user_id" => $user,
+                    "speaker_id" => $speakers->random(),
```
Comment (krzysztofrewak) [inline database/seeders/FrontendSeeder.php:36]: Are you sure that `random()` will return ID and not a Speaker instance?

Comment (MrHDOLEK) [reply database/seeders/FrontendSeeder.php:36]: yes and I tested this function for run seeders and random return random object speakers for collection

Comment (krzysztofrewak) [reply database/seeders/FrontendSeeder.php]: So I don't get it. You're asasigning an object to speaker id and it's okay?

File: database/seeders/FrontendSeeder.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+use Blumilk\Meetup\Core\Models\Meetup;
+use Blumilk\Meetup\Core\Models\Organization;
+use Blumilk\Meetup\Core\Models\Speaker;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Hash;
+
+class FrontendSeeder extends Seeder
```
Comment (kamilpiech97) [inline database/seeders/FrontendSeeder.php:15]:
```suggestion
class DummyDataSeeder extends Seeder
```
that will be more readable

File: environment/dev/php/bin/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -ea
+
+if [ ! -d "vendor" ]; then
+  echo "Vendor not installed. Installing..."
```
Comment (kamilpiech97) [inline environment/dev/php/bin/entrypoint.sh:5]:
```suggestion
  echo "Vendor does not exist. Installing..."
```

File: database/seeders/FrontendSeeder.php (hunk 2 above)
Comment (MrHDOLEK) [reply database/seeders/FrontendSeeder.php]: Speakers-> random return object sppeaker a laravel for this query assign object for relationship

File: database/seeders/DummyDataSeeder.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        Organization::factory(10)->create();
+        Speaker::factory(10)->create();
+        $speakers = Speaker::all();
+        $organization = Organization::all();
+        foreach ($speakers as $speaker) {
+            Meetup::factory(10)->create([
+                "user_id" => $user,
+                "speaker_id" => $speakers->random(),
+                "organization_id" => $organization->random(),
+            ]);
```
Comment (krzysztofrewak) [inline database/seeders/DummyDataSeeder.php:35]:
```suggestion
        ])->create();

        $organization = Organization::factory(10)->create();
        $speakers = Speaker::factory(10)->create();

        foreach ($speakers as $speaker) {
            Meetup::factory(10)->create([
                "user_id" => $user,
                "speaker_id" => $speakers->random(),
                "organization_id" => $organizations->random(),
            ]);
```

I think it should work too. Also: please take care about vertical density and proper naming of variables.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Review by MichalMyskow — APPROVED** [review]

**Pull request merged**